test/features/deploy_quarkus_service.feature

```diff
@@ -151,6 +151,7 @@ Feature: Deploy quarkus service
       | disabled | 10      |
 
     @native
+    @disabled
```

Comment:
Please add a comment here referencing the issue.

Reply:
added